accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -0,0 +1,400 @@
+        row["Max Rqst Cpus"]    = max(self.clean(data["RequestCpus"], allow_empty_list=False))
+        row["Num Exec Atts"]    = sum(self.clean(num_exec_attempts))
+        row["Num Shadw Starts"] = sum(self.clean(num_shadow_starts))
+        row["Num Local Univ Jobs"] = sum(data["_NumLocalUnivJobs"])
+        row["Num Sched Univ Jobs"] = sum(data["_NumSchedulerUnivJobs"])
+
+        # Compute derivative columns
+        if row["All CPU Hours"] > 0:
+            row["% Good CPU Hours"] = 100 * row["Good CPU Hours"] / row["All CPU Hours"]
+        else:
+            row["% Good CPU Hours"] = 0
+        if row["Num Uniq Job Ids"] > 0:
+            row["Shadw Starts / Job Id"] = row["Num Shadw Starts"] / row["Num Uniq Job Ids"]
+        else:
+            row["Shadw Starts / Job Id"] = 0
+        if row["Num Shadw Starts"] > 0:
+            row["Exec Atts / Shadw Start"] = row["Num Exec Atts"] / row["Num Shadw Starts"]
+        else:
+            row["Exec Atts / Shadw Start"] = 0
+        if sum(data["_NumBadJobStarts"]) > 0:
+            row["CPU Hours / Bad Exec Att"] = (sum(badput_cpu_time) / 3600) / sum(data["_NumBadJobStarts"])
+        else:
+            row["CPU Hours / Bad Exec Att"] = 0
+
+        # Compute mode for Project and Schedd columns in the Users table
+        if agg == "Users":
+            projects = self.clean(data["ProjectName"])
+            if len(projects) > 0:
+                row["Most Used Project"] = max(set(projects), key=projects.count)
+            else:
+                row["Most Used Project"] = "UNKNOWN"
+
+            schedds = self.clean(data["ScheddName"])
+            if len(schedds) > 0:
+                row["Most Used Schedd"] = max(set(schedds), key=schedds.count)
+            else:
+                row["Most Used Schedd"] = "UNKNOWN"
+        if agg == "Projects":
+            row["Num Users"] = len(set(data["User"]))
+
+        # Do whatever else here
+        if agg == "Users":
+            for i in range(len(data["_NumJobs"])):
+                if data["CommittedTime"][i] > 20*3600:
+                    user = agg_name
+                    jobid = data["GlobalJobId"][i]
+                    walltime = data["CommittedTime"][i]
+                    startdname = data["StartdName"][i]
+                    startdip = data["StartdPrincipal"][i].split("/")[-1]
+                    mips = data["MachineAttrMips0"][i] or "UNKNOWN"
+                    line = f"{startdip}\t{startdname}\t{mips}\t{walltime/3600:.1f}\t{user}\t{jobid}\n"
+                    long_job_log = Path.cwd() / "long_job_log.txt"
+                    if not long_job_log.exists():
+                        with long_job_log.open("w") as f:
+                            f.write("IP\tName\tMips\tHours\tUser\tGlobalJobId\n")
+                            f.write(line)
+                    else:
+                        with long_job_log.open("a") as f:
+                            f.write(line)
+
+        return row
```

Comment:
My hack for writing out a log of long jobs with mips values doesn't really apply here
```suggestion
```